### ISSUE TITLE
Runtime and jest warning/console error

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/jbrowse-components#907]
```
index.js:1 Warning: Cannot update a component (`SvgFeatureRendering`) while rendering a different component (`wrappedComponent`). To locate the bad setState() call inside `wrappedComponent`, follow the stack trace as described in https://fb.me/setstate-in-render
    in wrappedComponent
    in wrappedComponent (at SvgFeatureRendering.js:396)
    in svg (at SvgFeatureRendering.js:378)
    in div (at SvgFeatureRendering.js:377)
    in SvgFeatureRendering
    in SvgFeatureRendering
    in RenderErrorBoundary
```

Happens for alignment track and SVG features track

Also found this for closing a track, but may be a race condition

![Screenshot from 2020-05-04 17-21-37](https://user-images.githubusercontent.com/6511937/81014953-d1093180-8e2b-11ea-8407-56a97aac2158.png)

**Comment by garrettjstevens** [comment]
Looks like this is a new warning in React 16.3: https://reactjs.org/blog/2020/02/26/react-v16.13.0.html#warnings-for-some-updates-during-render

**Comment by cmdcolin** [comment]
The svgfeaturerendering one was fixed

There is another one that remains

If you open up hg19 and turn on an alignments track e.g. skbr3 splitters only it will popup this warning

The stack trace goes back to onSnapshot(session, ... setUrlSnapshot) in line 78 of JBrowse.js

**Comment by cmdcolin** [comment]
I believe remaining issues related to this will need to have some of the URL logic from jbrowse.js go into the model or similar, a template for this here

https://github.com/GMOD/jbrowse-components/tree/url_logic_in_model

**Comment by cmdcolin** [comment]
The two react 16.13 warnings were fixed via #976 and  abca58d37

The 'no session model found' was fixed via commit commit 0428a4d7b29
